plugins/org.jboss.tools.openshift.cdk.server/src/org/jboss/tools/openshift/cdk/server/core/internal/adapter/VagrantPoller.java

```diff
@@ -187,28 +189,22 @@ private boolean checkOpenShiftHealth(IServer server, int timeout) throws OpenShi
 		if( adb == null ) {
 			return false;
 		}
-		String url = adb.getOpenShiftHost() + ":" + adb.getOpenShiftPort() + "/healthz/ready";
+		String url = adb.getOpenShiftHost() + ":" + adb.getOpenShiftPort();
```

Comment:
Functionality moved into the client and is essentially here: https://github.com/jbosstools/jbosstools-openshift/pull/1239/commits/a021ad2ba8f1ce6f06bbfa452837b6104831750e#diff-29e83e4203e38d1faadbabcea0001398R204